atlasdb-dbkvs-tests/src/test/resources/docker-startup/01_create_user.sql

```diff
@@ -0,0 +1,4 @@
+CREATE USER palantir IDENTIFIED BY "7_SeeingStones_7";
+grant create session, create procedure, create sequence, create table, create view, create type to palantir;
```

Comment:
question: is this a template off somewhere / should we be consistent with the other commands in terms of casing?

Reply:
Updated with formatting